pvlib/clearsky.py

```diff
@@ -720,3 +720,103 @@ def rmse(alpha):
         return clear_samples, components, alpha
     else:
         return clear_samples
+
+
+def bird(zenith, airmass_relative, aod380, aod500, precipitable_water,
+         ozone=0.3, pressure=101325., dni_extra=1364., asymmetry=0.85,
+         albedo=0.2):
+    """
+    NREL Bird Simple Clear Sky Broadband Solar Radiation Model
+
+    Based on original implementation by Daryl R. Myers at NREL.
+
+    Parameters
+    ----------
+    zenith : numeric
+        Solar zenith angle in degrees.
+    airmass_relative : numeric
+        Relative airmass.
+    aod380 : numeric
+        Aerosol optical depth [cm] measured at 380[nm].
+    aod500 : numeric
+        Aerosol optical depth [cm] measured at 500[nm].
+    precipitable_water : numeric
+        Precipitable water [cm]
+    ozone : numeric
+        Atmospheric ozone [cm], defaults to 0.3[cm]
+    pressure : numeric
+        Ambient pressure [Pa], defaults to 101325[Pa]
+    dni_extra : numeric
+        Extraterrestrial radiation [W/m^2], defaults to 1364[W/m^2]
+    asymmetry : numeric
+        Asymmetry factor, defaults to 0.85
+    albedo : numeric
+        Albedo, defaults to 0.2
+
+    Returns
+    -------
+    direct_beam : numeric
```

Comment:
It's more conventional in pvlib to return a DataFrame or OrderedDict, rather than a tuple. Series in, DataFrame out, anything else in, OrderedDict out. The other functions in this module demonstrate this pattern. columns/keys should follow http://pvlib-python.readthedocs.io/en/latest/variables_style_rules.html